### PR TITLE
Task 2 Create a design system, button and input stylized

### PR DESCRIPTION
**Pull request by NondasNF**
Peek 11-02-2022 03-55

### CODE REVIEW

File: src/design-system.html [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap" rel="stylesheet">
+  <title>Design System</title>
+</head>
+
+<body>
+  <h1>Design System</h1>
+  <div class="buttons">
+    <button class="button__custom">Button</button>
```
Comment (samflores) [inline src/design-system.html]: You are still not applying BEM naming schema properly. 
`__` should used to define a element name (an elemente inside a block)
`--` should be used to define modifiers

File: src/styles/components/_button.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  cursor: pointer;
+}
+.button__custom:active{
+  background: var(--yellow-2);
+}
+.button__custom:disabled{
+  filter: grayscale(60%);
+  background: var(--yellow-1);
+  cursor:default;
+}
```
Comment (samflores) [inline src/styles/components/_button.css:21]: The challenge asked you to define variations for buttons (yellow and blue).

File: src/styles/components/_text-input.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.input__text{
```
Comment (samflores) [inline src/styles/components/_text-input.css]: Same thing about variations and properly separating blocks, elements and modifiers.

**Comment by samflores** [comment]
You could try to match the look of the screenshot given in the challenge a bit more. The spacing and color don't match.

File: src/styles/components/_button.css (hunk 2 above)
Comment (NondasNF) [reply src/styles/components/_button.css:21]: On a first look I thought it was about the normal button and the same button disabled, thanks

**Pull request merged**